scripts/migration/GPII-4014/README.md

```diff
@@ -0,0 +1,27 @@
+# Data Migration and Verification for GPII-4014 Deployment
```

Comment:
I think it would be helpful if this directory name also incorporated the date, schema or some other sequence number information about the migration since once we accumulate a lot of these it might be hard to determine which migrations happened first. 
I suggest calling it something like schema-0.2-GPII-4014, 20190731-GPII-4014 or something similar